components/newsletter-sign-up/newsletter-sign-up.jsx

```diff
@@ -272,7 +272,7 @@ class NewsletterSignUp extends React.Component {
 
 NewsletterSignUp.defaultProps = {
   ctaHeader: `Protect the internet as a global public resource`,
-  ctaDescription: `Join our email list to take action and stay updated!`,
+  ctaDescription: `Join our Mozilla News email list to take action and stay updated!`,
```

Comment:
I would change line 137-141 to 
```jsx
<div
  dangerouslySetInnerHTML={{
    __html: this.props.ctaDescription,
  }}
/>
```
and this line to
```suggestion
  ctaDescription: `<p>Join our <b>Mozilla News</b> email list to take action and stay updated!</p>`,
```